Scala/Scala.sublime-syntax

```diff
@@ -7,6 +7,9 @@ file_extensions:
   - sbt
 scope: source.scala
 variables:
+  # lookahead/behind for operators
+  oplookbehind: '(?<=[[[:alpha:]]0-9\s\(\)\[\]\{\}])'
```

Comment:
Well, the problem is that those operators were just random examples that I constructed on the fly.  Scala allows *arbitrary* symbolic operators (truly arbitrary).  The only restriction is that a couple operators are reserved.  Single-character `:` and `=>` are among them.  This is literally the symbolic analogue to keywords (e.g. `def` is a valid identifier, but the keyword takes precedence).  The problem is that we can't leverage `\b` to pull out symbolic keywords ahead of time, which is why I needed the backtracking in the first place.

In general, I hate backtracking.  I've done a lot of academic work in the field of parser theory, so I'm very familiar with how horrible it is.  :-)  I just couldn't find another solution in this particular case.